src/pages/Main/BtnStart/BtnStart.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './BtnStart.scss';
+
+class BtnStart extends React.Component {
+  render() {
+    return <button className="btnStart">지금 시작하기</button>;
```

Comment:
```suggestion
    return <button className="BtnStart">지금 시작하기</button>;
```
최상위 className은 컴포넌트 이름과 동일하게, cacelCase 말고 Pascal로 적어주시면 됩니다!
전체 작업에서 동일한 부분있으면 다 수정해주세요

Reply:
이 머지 안된 브랜치에는 일괄적으로 반영하겠습니다. 
머지 된 브랜치는 기능 구현하면서 해당 브랜치에서 변경하겠습니다!